model/src/main/kotlin/VcsInfo.kt

```diff
@@ -83,6 +83,11 @@ data class VcsInfo(
      * Return a [VcsInfoCurationData] with the properties from this [VcsInfo].
      */
     fun toCuration() = VcsInfoCurationData(type, url, revision, path)
+
```

Comment:
Commit message nits:
- Duplicate "method" in first sentence.
- Please use passive voice, i.e. no "we", "I", "us" etc.

Reply:
overhauled the commit message after the changes to the commit.

Reply:
Pushed new commits with code changes only.
I will squash them once they are reviewed and accepted.
Afterwards I will update the commit messages and force push.

Reply:
Just pushed the squashed commits with the overhauled messages.